### PR TITLE
Update

### PR DESCRIPTION
**Pull request by J22Melody**

### CODE REVIEW

File: cldfbench_tonodb.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def glang_attrs(glang, languoids):
 class Dataset(BaseDataset):
     dir = pathlib.Path(__file__).parent
     id = "tonodb"
-    valueTableProperties = ['LanguageVariety', 'TriggeringContext' , 'Extra', 'Height', 'Contour', 'Phonation', 'ToneDescription', 'ChaoNumerals', 'Notes', 'EffectOnPitch', 'ResultantSystem', 'Type', 'Onset', 'Coda', 'Stress/quantity', 'Wordtype', 'long vowel']
+    # valueTableProperties = ['LanguageVariety', 'TriggeringContext' , 'Extra', 'Height', 'Contour', 'Phonation', 'ToneDescription', 'ChaoNumerals', 'Notes', 'EffectOnPitch', 'ResultantSystem', 'Type', 'Onset', 'Coda', 'Stress/quantity', 'Wordtype', 'long vowel']
+    valueTableProperties = ['LanguageVariety', 'TriggeringContext', 'Extra', 'Height', 'Contour', 'Phonation', 'ToneDescription' , 'ChaoNumerals', 'Notes', 'EffectOnPitch', 'ResultantSystem', 'Type', 'Onset', 'Coda', 'Stress/quantity', 'Wordtype', 'Nucleus']
```
Comment (J22Melody) [inline cldfbench_tonodb.py:34]: I believe here we changed a column from `'long vowel'` to `'Nucleus'`, could you confirm? @bambooforest @uzling?

Comment (bambooforest) [reply cldfbench_tonodb.py:34]: @J22Melody -- I think it was supposed to be Nucleus before with long vowel as one of its options, so this fixes that. Thx!

**Comment by bambooforest** [comment]
@J22Melody @xrotwang -- build (Python) 3.6 failed -- is that an issue?

**Comment by xrotwang** [comment]
@bambooforest fixed: https://github.com/cldf-datasets/tonodb/commit/28fd651eae0b03194c7b822d26312240771011be
Python 3.6 is EOL and no longer supported by most of my packages.

**Pull request merged**